### PR TITLE
#1480: Shortened title on smaller screens

### LINKED ISSUE #1480: Fix 'Finishline BY NER' title looking crappy on small screens
### Description

Right now the 'Finishline By Ner' Title overflows on mobile or if you make the screen too small. This ticket should be too make the title shrink to just be 'Finishline' or not even be displayed at all if the screen is too small.

### Acceptance Criteria

- The top navbar doesn't look crappy when you make the screen too small

### Proposed Solution

N/A

### Mocks

img_6950_720

### PR DESCRIPTION
**Pull request by mattrwang**
## Changes

Got rid of "by NER" in the title when on mobile/smaller windows.

## Screenshots

image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #1480

### CODE REVIEW

**Review by walker-sean — COMMENTED** [review]
Looks good!

@RChandler234 do we want this to be responsive or is it good this way?

**Review by walker-sean — CHANGES_REQUESTED** [review]
Can you make this responsive - as in, when the browser changes size, the title will change?

Also, that 550 is a "magic constant"/"magic number" which makes code harder to understand. Try to figure out a way to avoid using this number. If needed, give this constant a name so it's clear what the number is.

File: src/frontend/src/layouts/NavTopBar/NavTopBar.tsx [hunk 1]
```diff
@@ -15,15 +15,24 @@ import { LayoutProps } from '../LayoutProps';
...
+const mobileWidthThreshold = 550;
 
 interface NavTopBarProps extends LayoutProps {
   handleDrawerOpen: () => void;
 }
 
 const NavTopBar: React.FC<NavTopBarProps> = ({ open, handleDrawerOpen }) => {
+  const [width, setWidth] = React.useState(window.innerWidth);
+  React.useEffect(() => {
+    function handleResize() {
```
Comment (walker-sean) [inline src/frontend/src/layouts/NavTopBar/NavTopBar.tsx:31]: Make this an arrow function (using `=>`) inside the call to `addEventListener`

File: src/frontend/src/layouts/NavTopBar/NavTopBar.tsx [hunk 2]
```diff
@@ -15,15 +15,24 @@ import { LayoutProps } from '../LayoutProps';
 import { IconButton } from '@mui/material';
 import { GridMenuIcon } from '@mui/x-data-grid';
 import { useCurrentUser } from '../../hooks/users.hooks';
+import React from 'react';
```
Comment (walker-sean) [inline src/frontend/src/layouts/NavTopBar/NavTopBar.tsx:18]: Change to `import { useEffect, useState } from 'react';`

File: src/frontend/src/layouts/NavTopBar/NavTopBar.tsx [hunk 3]
```diff
@@ -15,15 +15,24 @@ import { LayoutProps } from '../LayoutProps';
...
 const background = '#ef4345';
+const mobileWidthThreshold = 550;
 
 interface NavTopBarProps extends LayoutProps {
   handleDrawerOpen: () => void;
 }
 
 const NavTopBar: React.FC<NavTopBarProps> = ({ open, handleDrawerOpen }) => {
+  const [width, setWidth] = React.useState(window.innerWidth);
+  React.useEffect(() => {
```
Comment (walker-sean) [inline src/frontend/src/layouts/NavTopBar/NavTopBar.tsx:30]: Remove `React.`

File: src/frontend/src/layouts/NavTopBar/NavTopBar.tsx [hunk 4]
```diff
@@ -15,15 +15,24 @@ import { LayoutProps } from '../LayoutProps';
...
 const textColor = 'white';
 const background = '#ef4345';
+const mobileWidthThreshold = 550;
 
 interface NavTopBarProps extends LayoutProps {
   handleDrawerOpen: () => void;
 }
 
 const NavTopBar: React.FC<NavTopBarProps> = ({ open, handleDrawerOpen }) => {
+  const [width, setWidth] = React.useState(window.innerWidth);
```
Comment (walker-sean) [inline src/frontend/src/layouts/NavTopBar/NavTopBar.tsx:29]: Remove `React.`

**Review by walker-sean — CHANGES_REQUESTED** [review]
Looks great! Love having it reactive like this. Just a few things to fix up

**Review by walker-sean — DISMISSED** [review]
LGTM!

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**